app/forms/instance-create.tsx

```diff
@@ -388,29 +430,57 @@ export function CreateInstanceForm() {
             />
           )}
         </Tabs.Content>
+
+        <Tabs.Content value="disk" className="space-y-4">
+          {disks.length === 0 ? (
+            <div className="flex max-w-lg items-center justify-center rounded-lg border p-6 border-default">
+              <EmptyMessage
+                icon={<Storage16Icon />}
+                title="No detached disks found"
+                body="A disk needs to be created and not attached to an instance to be used as a boot disk"
```

Comment:
I get where this comes from, but "A disk needs to be created" is a bit computer-brained. Gonna tweak it to this

<img width="553" alt="image" src="https://github.com/oxidecomputer/console/assets/3612203/96ae5619-a40e-419b-8c6b-f8d5a0bed61e">